packages/cli/fragments/fragment-clojurescript/src/main/clojure/app/core.cljs

```diff
@@ -0,0 +1,25 @@
+(ns app.core
+  (:require
+   [goog.dom :as gdom]
+   [reagent.dom :as dom]))
+
+
+(defn square
+  [x]
+  (* x x))
+
+
+(defn mount-root
+  "Mount root component."
+  {:dev/after-load true}
+  []
+  (some->>
+   (gdom/getElement "root")
+   (dom/render [:h1 "Welcome to Tauri!"])))
```

Comment:
This is a very very basic example, there should be an example showcasing how to invoke a rust command using https://tauri.app/v1/api/js/tauri#invoke

Also you need to match the same structure and functionality of the other templates which uses the css from `base/src/styles.css` and have a simple example for invoking rust command